File: 5-kyu/simple-pig-latin/ruby/solution.rb
def pig_it(str)
  alpha = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
  res = str.split(" ").map { |word|
    if (alpha.split("").include?(word.split("").first))
      word = (word + word.split("").first + "ay")
      word = word[1..-1]
    end
    word
  }
  return res.join(" ")
end

File: 5-kyu/simple-pig-latin/javascript/solution.js
var alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
function pigIt(str){
  console.log("To split: " + str)
  return str.split(" ").map(word => {
    if (alphabet.includes(word[0]))
    {
      return word.substr(1) + [...word][0] + "ay";
    }
    return word
  }).join(" ")
}